docs/source/cn/cuda.py

```diff
@@ -0,0 +1,71 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.cuda.BoolTensor,
+    r"""
+    创造一个 bool 类型张量，其和 :func:`oneflow.Tensor` 共享相同的参数。
```

Comment:
```suggestion
    创造一个 bool 类型张量，它的其余参数与 :func:`oneflow.Tensor` 一致。
```

是不是改成这种句式更好？若接受，本PR的其它地方请同步修改了。

Reply:
好的